### PR TITLE
Add error message to ansible2john

### PR DESCRIPTION
**Pull request by marcan2020**

### CODE REVIEW

File: run/ansible2john.py [hunk 1]
```diff
@@ -31,6 +31,7 @@ def process_file(filename):
 
     data = open(filename, "rb").read()
     if not data.startswith(HEADER):
+        sys.stderr.write("File doesn't start with %s" % HEADER)
```
Comment (solardiz) [inline run/ansible2john.py]: Perhaps add `\n` after `%s`?

**Review by solardiz — CHANGES_REQUESTED** [review]

File: run/ansible2john.py (hunk 1 above)
Comment (marcan2020) [reply run/ansible2john.py]: @solardiz Good call. I changed it.

**Comment by solardiz** [comment]
@marcan2020 Can you please squash the two commits into one and force-push? Alternatively, let us know that we should use GitHub's "Squash and merge", but it's dirtier (makes GitHub the committer per git history). Thanks!

**Comment by marcan2020** [comment]
@solardiz It's done.

**Review by solardiz — APPROVED** [review]

**Comment by solardiz** [comment]
@marcan2020 Merged. Thank you!

**Pull request merged**